Doc/library/inspect.rst

```diff
@@ -526,12 +527,14 @@ Retrieving source code
 
 .. function:: getmodule(object)
 
-   Try to guess which module an object was defined in.
+   Try to guess which module an object was defined in. Return ``None``
+   in case of an error.
 
 
 .. function:: getsourcefile(object)
 
-   Return the name of the Python source file in which an object was defined.  This
+   Return the name of the Python source file in which an object was defined
+   or ``None`` if no way can be identified to get the source.  This
```

Comment:
```suggestion
   or ``None`` if it cannot be identified.  This
```

Reply:
Thanks! But, I've copied this exact piece from `inspect.py`, I think it is better to keep it in sync (at least try to do that) 🙂